meta-mbl-distro/scripts/create-update-payload

```diff
@@ -72,23 +73,28 @@ bitbakepath = scriptpath.add_bitbake_lib_path()
 import bb.tinfoil
 import argparse_oe
 
+import pdb
+MBL_BOOT_ID = "MBL_BOOT"
+PAYLOAD_FORMAT_VERSION = b"1"
+
 
 def parse_args():
     """Parse the command line arguments."""
     parser = argparse_oe.ArgumentParser(description=__doc__)
-    group = parser.add_mutually_exclusive_group(required=True)
-    group.add_argument(
-        "-f",
-        "--filenames",
-        nargs="+",
-        help="Space separated list of filenames of update components.",
-    )
-    group.add_argument(
+    parser.add_argument(
         "-b",
-        "--bootloader-slots",
+        "--bootloader-components",
         nargs="+",
-        help="Space separated list of bootloader slot numbers. e.g 1 would"
-        " correspond to bootloader slot one.",
+        help="Space separated list of bootloader component numbers. e.g 1 would"
+        " correspond to bootloader component 1.",
+    )
+    parser.add_argument(
+        "-f",
```

Comment:
Why is this `-f` rather than e.g. `-k`?

Reply:
Addressed in https://github.com/ARMmbed/meta-mbl/pull/704/commits/da9a2b3022d429c15a8bbe0ec277e2d24a98fbf9.